scripts/upload_artifact_s3.py

```diff
@@ -0,0 +1,38 @@
+import os
```

Comment:
if you add shebang, you can make this script executable

Reply:
To be honest I don't know if we want it executable. Since we're expected to only use this in CI, I think we can just be calling it from within our code and we avoid using subprocess which I feel is very unsafe to be running.

Reply:
that's ok then